doc/source/release/1.11.0-notes.rst

```diff
@@ -31,68 +31,220 @@ Highlights of this release
+- For an even number of points, `scipy.integrate.simpson` now calculates
+  a parabolic segment over the last three points which gives improved
+  accuracy over the previous implementation.
+
 `scipy.cluster` improvements
 ============================
+- ``disjoint_set`` has a new method ``subset_size`` for providing the size
+  of a particular subset.
 
 
 `scipy.interpolate` improvements
 ================================
 
+`scipy.constants` improvements
+================================
+- The ``quetta``, ``ronna``, ``ronto``, and ``quecto`` SI prefixes were added.
+
 
 `scipy.linalg` improvements
 ===========================
+- `scipy.linalg.det` is improved and now accepts nD-arrays.
+- `scipy.linalg.lu` is improved and now accepts nD-arrays. With the new
+  ``matrix_p`` switch the output permutation argument can be 1D ``(n,)``
+  permutation index instead of the full ``(n, n)`` array.
 
 
 `scipy.ndimage` improvements
 ============================
+- ``axes`` argument was added to ``rank_filter``, ``percentile_filter``,
+  ``median_filter``, ``uniform_filter``, ``minimum_filter``,
+  ``maximum_filter``, and ``gaussian_filter``, which can be useful for
+  processing stacks of image data.
 
 
 `scipy.optimize` improvements
 =============================
+- `scipy.optimize.linprog` now passes unrecognized options directly to HiGHS.
+- `scipy.optimize.root_scalar` now uses Newton's method to be used without
+  providing ``fprime`` and the ``secant`` method to be used without a second
+  guess.
+- `scipy.optimize.lsq_linear` now accepts ``bounds`` arguments of type
+  `scipy.optimize.Bounds`.
+- `scipy.optimize.minimize` ``method='cobyla'`` now supports simple bound
+  constraints.
+- Users can opt into a new callback interface for most methods of
+  `scipy.optimize.minimize`: If the provided callback callable accepts
+  a single keyword argument, ``intermediate_result``, `scipy.optimize.minimize`
+  now passes both the current solution and the optimal value of the objective
+  function to the callback as an instance of `scipy.optimize.OptimizeResult`.
+  It also allows the user to terminate optimization by raising a
+  ``StopIteration`` exception from the callback function.
+  `scipy.optimize.minimize` will return normally, and the latest solution
+  information is provided in the result object.
+- `scipy.optimize.curve_fit` now supports an optional ``nan_policy`` argument.
+- `scipy.optimize.shgo` now has parallelization with the ``workers`` argument,
+  symmetry arguments that can improve performance, class-based design to
+  improve usability, and generally improved performance.
 
 
 `scipy.signal` improvements
 ===========================
-
+- ``istft`` has an improved warning message when the NOLA condition fails.
 
 `scipy.sparse` improvements
 ===========================
+- `scipy.sparse` array (not matrix) classes now return a sparse array instead
+  of a dense array when divided by a dense array.
+-
```

Comment:
```suggestion
- A new public base class `scipy.sparse.sparray` was introduced, allowing `isinstance(x, scipy.sparse.sparray)` to select the new sparse array classes, while `isinstance(x, scipy.sparse.spmatrix)` selects only the old sparse matrix types.
- The behavior of `scipy.sparse.isspmatrix()` was updated to return True for only the sparse matrix types. If you want to check for either sparse arrays or sparse matrices, use `scipy.sparse.issparse()` instead. (Previously, these had identical behavior.)
- Several methods were deprecated for sparse arrays: `asfptype`, `getrow`, `getcol`, `getnnz`, and `getformat`. Additionally, the `.A` and `.H` attributes were deprecated. Sparse matrix types are not affected.
- Sparse arrays constructed with 64-bit indices will no longer automatically downcast to 32-bit.
- A new `scipy.sparse.diags_array` function was added, which behaves like the existing `scipy.sparse.diags` function except that it returns a sparse array instead of a sparse matrix.
- `argmin` and `argmax` methods now return the correct result when no implicit zeros are present.
```